### DISCUSSION TITLE
AMD 5700XT oc optimisation

### DISCUSSION THREAD

**Discussion by shamonis** [Lolliedieb/lolMiner-releases#208] [Q&A] [answered]
Hi team,

running a stock AMD 5700XT. 
Using OC settings:
Core -> 1500MHz, 900mV
VRAM -> 1800MHz

I am not getting even close to 50Mh/s as expected without modding the bios. (Getting around 20Mh/s)
![image](https://user-images.githubusercontent.com/24393262/106593585-e6d98d80-6548-11eb-9242-42795e345e81.png)
Any obvious problems?

Also even if I have a single GPU, lolMiner reports 2 devices:
![image](https://user-images.githubusercontent.com/24393262/106592816-e2f93b80-6547-11eb-85ea-f81e22aefb67.png)

Thanks in advance!

**Comment by croelanjr** [comment]
Hi, If you are mining on Windows. You have to activate compute mode. Or you must modify the values in the GPU bios.

**Comment by jh442** [comment]
1500 core clock seems like a lot as well.  Use the afterburner software and try 1350/1800 combo.  Bios modding is fairly straight forward to and recommend saving your original and trying that out.  No reason to get any less then 52 really with a 5700

**Comment by madrunner91** [comment, marked as answer]
Here is what I'm using as an over clock on my Red Devil 5700XT, using 20.12.1. Getting around 51mh/s
![5700xtOverclock](https://user-images.githubusercontent.com/78997646/107854786-f035e600-6de3-11eb-96bc-d96b42ad947b.PNG)
![hashrate](https://user-images.githubusercontent.com/78997646/107854790-f4fa9a00-6de3-11eb-9510-140852e672d1.png)
Ignore the other gpu it's my 580

**Reply by shamonis** [reply]
Thanks a lot for the reply! Are you also using  --computemode?

**Reply by madrunner91** [reply]
I actually am not. On the 580 in radeon settings I have compute mode on. There is no option for compute mode on the 5700 so I'm not running it.
